docs/source/model_design/risk_factors.rst

```diff
@@ -195,7 +195,68 @@ relationships directionally between a cause/exposure and effect/outcome.
   Discuss criteria (sim science and GBD) for modeling a risk outcome pair
 
 Confounding
-""""""""""""
+"""""""""""
+**What is confounding?**
```

Comment:
1) first define confounding
2) then describe RCTs vs observational studies and the role of confounding in each.